### PR TITLE
Add LICENSE/NOTICE.md to sqlcmd install folder

### PR DESCRIPTION
**Pull request by stuartpa**
1. Add LICENSE/NOTICE.md to the install directory on Windows .msi installs (winget/choco)
2. Add arm/arm64 .msi builds (a separate winget repo PR adds these to the winget installer)
3. Also, align Architecture naming to winget/brew/debian:

So we will have:

x64 (for Windows/Mac)
amd64 (for Linux)

x86 (for Windows/Mac)
arm (for all Platforms)
arm64 (for all Platforms)

Sources:

https://github.com/jedieaston/winget-cli/blob/0384fcd3657942f59bc94aae3dd888742f9ef632/schemas/JSON/settings/settings.schema.0.2.json#L58

            "enum": [
              "neutral",
              "x64",
              "x86",
              "arm64",
              "arm"
            ],

This also seems to be align with brew:

https://github.com/Homebrew/homebrew-core/blob/361429d75bf64c698c3d971f7aeebe01cedb87c2/Formula/sqlcmd.rb

    sha256 cellar: :any_skip_relocation, arm64_ventura:  
    sha256 cellar: :any_skip_relocation, arm64_monterey:
    sha256 cellar: :any_skip_relocation, arm64_big_sur:  
    sha256 cellar: :any_skip_relocation, ventura:        
    sha256 cellar: :any_skip_relocation, monterey:       
    sha256 cellar: :any_skip_relocation, big_sur:        
    sha256 cellar: :any_skip_relocation, x86_64_linux:   
    
For Debian installers, the choices are:

Available options: amd64,arm64,armel,armhf,i386,mips,mips64el,mipsel,ppc64el,s390x

### CODE REVIEW

File: build/azure-pipelines/build-product.yml [hunk 1]
```diff
@@ -159,7 +159,7 @@ stages:
           rootFolderOrFile: '$(Pipeline.Workspace)\SqlcmdWindowsAmd64'
           includeRootFolder: false
           archiveType: 'zip'
-          archiveFile: '$(Build.ArtifactStagingDirectory)/sqlcmd-$(getVersion.VERSION_TAG)-windows-x64.zip'
+          archiveFile: '$(Build.ArtifactStagingDirectory)/sqlcmd-$(getVersion.VERSION_TAG)-windows-amd64.zip'
```
Comment (shueybubbles) [inline build/azure-pipelines/build-product.yml:162]: >-amd64.zip' [](http://example.com/codeflow?start=98&length=11)

Other repos depend on this name 
see [https://github.com/Azure/sql-action/blob/2177a5b5bb910608e2a3d10140dc8b604d68f10f/src/Setup.ts#L38](https://github.com/Azure/sql-action/blob/2177a5b5bb910608e2a3d10140dc8b604d68f10f/src/Setup.ts#L38)

File: cmd/modern/root.go [hunk 2]
```diff
@@ -119,6 +119,7 @@ func (c *Root) addGlobalFlags() {
 		DefaultString: config.DefaultFileName(),
 		Name:          "sqlconfig",
 		Usage:         "configuration file",
+		Persistent:    true,
```
Comment (shueybubbles) [inline cmd/modern/root.go:122]: >sistent:    true, [](http://example.com/codeflow?start=5&length=17)

are the changes in .go files meant to be part of this PR?

File: build/azure-pipelines/build-product.yml (hunk 1 above)
Comment (shueybubbles) [reply build/azure-pipelines/build-product.yml:162]: it's probably fine to change it just be sure to file an issue on sql-action at the same time to update their usage

Comment (stuartpa) [reply build/azure-pipelines/build-product.yml]: I've changed it back to x64.  This area is very confusing.  (what label to use for each architecture, I've updated the PR comment)

**Review by JyotikaGargg — APPROVED** [review]

**Pull request merged**